app/models/manageiq/providers/ansible_tower/automation_manager/machine_credential.rb

```diff
@@ -1,2 +1,6 @@
 class ManageIQ::Providers::AnsibleTower::AutomationManager::MachineCredential < ManageIQ::Providers::AutomationManager::Authentication
+  ATTRIBUTES = [:description, :username, :ssh_key_data, :ssh_key_unlock,
+    :become_method, :become_username, :become_password, :vault_password].freeze
```

Comment:
Prefer enumerating one per line and list alphabetically.  This way new entires do not shake up the diff too much.